README.md

```diff
@@ -1,32 +1,34 @@
 # Solargraph::Rails - Help solargraph with Rails
 
 ## Models
-Given a typical Rails model like this:
+Consider pair of typical Rails models like this:
+
+```sh
+rails g model Author lastname:string firstnames:string
+rails g model Book title:string isbn:string author:belongs_to
+```
 
 ```ruby
-# == Schema Information
-#
-# Table name: my_books
-#
-#  id         :integer          not null, primary key
-#  author     :string
-#  name       :string
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
-#
```

Comment:
Very nice to show off the association support, but I wonder if we should also show off the annotate gem support for non-association columns